exiftool.go

```diff
@@ -218,6 +288,17 @@ func ExtractAllBinaryMetadata() func(*Exiftool) error {
 	}
 }
 
+// OverwriteOriginal overwrites the original file when writing the file metadata
+// instead of keeping a copy of the original (activates Exiftool's '-overwrite_original' parameter)
+// Sample :
+//   e, err := NewExiftool(OverwriteOriginal())
+func OverwriteOriginal() func(*Exiftool) error {
```

Comment:
I think that this option should be activated by default.

Judging from Exiftool's documentation, if you add metadata to `a.jpg` without `-overwrite_original`, then the original will be renamed `a.jpg_original`. That seems really really not natural to me.

On my mind, it should be overridden by default and if the user wants to keep the original, then:
- he backups himself the original
- or he add an option that disables `-overwrite_original`

I think that users will want to override file almost all the time. We should make it as simple and as direct as possible

Reply:
Since this library is a wrapper around `exiftool` I thought it was best to use `exiftool`'s default behavior. I *think* writing exif data is tricky, so `exiftool` makes a backup by default. It's your library, so it's your call how tightly coupled you want to be with `exiftool` and what ergonomics you'd like to change.

Reply:
Mhhhh, I honestly prefer my proposal.

Reply:
So would you like me to default the `Exiftool` struct to specify the `-overwrite_original` option by default and create a `KeepOriginal()` option?

Reply:
> So would you like the `Exiftool` struct to specify the `-overwrite_original` option by default and create a `BackupOriginal()` option?

Yes, that would be great